### PR TITLE
Add an article for the SR2021 knockouts

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes https://github.com/srobo/tasks/issues/668

### CODE REVIEW

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+started off with very different tactics, they were both accruing points very
+quickly. Due to a large number of link breakages, the points and leader switched
+massively throughout the match, but when the clock struck zero, it was 36 points
+to 8 for St Pauls, giving them the title.
+
+<!-- Video of Final -->
+TODO: make this embed not be the whole stream.
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/gxJM6ernMqo"
```
Comment (RealOrangeOne) [inline _posts/2021-05-01-sr2021-knockouts.md:70]: Match cut is live https://www.youtube.com/watch?v=Jwk2kUp02lk

```suggestion
<!-- Video of Final -->
<iframe
  class="center video"
  src="https://www.youtube-nocookie.com/embed/Jwk2kUp02lk"
```

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+During the league sessions there were some teams which showed strong early
+performances. Initially the teams from Hills Road Sixth Form College, looked to
+be the teams to beat. The later leagues however were the battleground of a
+different collection of teams, with KEGS Chelmsford (KEG), Gymnasium Markt
+Indersdorf (MAI) as well as rookie teams St Paul’s College (SPA) and We Robot
+(WER) in particular performing the best in the fourth and most complex of the
+leauge sessions.
+
+The Final
```
Comment (PeterJCLaw) [inline _posts/2021-05-01-sr2021-knockouts.md:56]: Thoughts on extending this into a fuller summary of the knockouts?

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+           caption="The podium positions" %}
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1<sup>st</sup> Place  | [SPA: St Pauls College, Adelaide](https://www.stpauls.sa.edu.au/)
+| 2<sup>nd</sup> Place  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 3
+| 3<sup>rd</sup> Place  | [The Malay College Kuala Kangsar](https://mckk.edu.my)
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 2
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [We Robot](https://www.werobot.fr)
+| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)
```
Comment (RealOrangeOne) [inline _posts/2021-05-01-sr2021-knockouts.md:96]: Use link sourced from team

```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.yshhb.org.bn/)
```

Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md]: For some reason GitHub wouldn't let me provide this via the UI: 8d4c2bf

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+           caption="The podium positions" %}
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1<sup>st</sup> Place  | [SPA: St Pauls College, Adelaide](https://www.stpauls.sa.edu.au/)
+| 2<sup>nd</sup> Place  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 3
+| 3<sup>rd</sup> Place  | [The Malay College Kuala Kangsar](https://mckk.edu.my)
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 2
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [We Robot](https://www.werobot.fr)
+| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.yshhb.org.bn/)
```
Comment (RealOrangeOne) [inline _posts/2021-05-01-sr2021-knockouts.md:96]: Turns out we were right before...

```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)
```

Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md:96]: :disappointed:

Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md]: e5bd53cbb7929afba78952e62da5b61b5fdf4b6c

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+the same.
+
+{% include figure.html
+           class="right"
+           link_to_image="true"
+           src="/images/content/news/sr2021/league-4-top-teams.png"
+           caption="Teams’ game points per league session." %}
+
+During the league sessions there were some teams which showed strong early
+performances. Initially the teams from Hills Road Sixth Form College, looked to
```
Comment (RealOrangeOne) [inline _posts/2021-05-01-sr2021-knockouts.md:49]:
```suggestion
performances. Initially the teams from Hills Road Sixth Form College looked to
```

Comment (RealOrangeOne) [reply _posts/2021-05-01-sr2021-knockouts.md:49]: Probably worth putting their TLA in too, to follow the pattern below

```suggestion
performances. Initially the teams from Hills Road Sixth Form College (HRS1-4) looked to
```

File: _posts/2021-05-01-sr2021-knockouts.md (hunk 2 above)
Comment (RealOrangeOne) [reply _posts/2021-05-01-sr2021-knockouts.md:73]: Is this now covered by the above "Knockouts" section?

File: _posts/2021-05-01-sr2021-knockouts.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+-------------------
+
+If you’d like to relive the highs and lows of the competition livestreams, the
+videos remain available on YouTube, as well as a cut-down video of the final:
+
+- [League Session 1](https://www.youtube.com/watch?v=cAvk-nfTUis)
+- [League Session 2](https://www.youtube.com/watch?v=RwW5Oz30gbE)
+- [League Session 3](https://www.youtube.com/watch?v=MpuhtW4mCKM)
+- [League Session 4](https://www.youtube.com/watch?v=E9gF-GZbf5M)
+- [Knockouts and Final](https://www.youtube.com/watch?v=gxJM6ernMqo)
```
Comment (RealOrangeOne) [inline _posts/2021-05-01-sr2021-knockouts.md:150]: Given this is a post about the knockouts and finals, should we embed this as an actual player?

Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md:150]: We already have one embed here and this article is pretty long (as is the livestream). I think we're fine to just leave this as a link.

File: _posts/2021-05-01-sr2021-knockouts.md (hunk 2 above)
Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md:73]: Yes.

File: _posts/2021-05-01-sr2021-knockouts.md (hunk 5 above)
Comment (PeterJCLaw) [reply _posts/2021-05-01-sr2021-knockouts.md]: Comma removed. I'm not sure the TLAs adds much here?

**Review by RealOrangeOne — APPROVED** [review]

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**